tests/functional_tests/autolab.sh

```diff
@@ -44,7 +51,7 @@ kill -SIGKILL "$msPid" > /dev/null 2>&1
 cd ../../main_server
 node main_server.js >>/tmp/log/main_server.log 2>&1 &
 msPid="$!"
-cd ../tests/functional_tests/
+cd ../tests/test_modules/
```

Comment:
putting this command after sleep, along with `bats scoreboard.bats` may aid readability.